spectral/instantiation/validation-rules.yaml

```diff
@@ -88,4 +89,12 @@ rules:
     message: "{{error}}"
     given: "$.nodes[*].unique-id"
     then:
-      function: node-has-relationship
\ No newline at end of file
+      function: node-has-relationship
+
+  unique-ids-must-be-unique:
```

Comment:
can we rename this to something like `unique-ids-must-be-unique-in-instantiation`? So we can keep it consistent with the naming convention of the other rules